includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2128,6 +2128,11 @@ public function get_level3_data_from_order( WC_Order $order ): array {
 		};
 		$items_to_send = array_map( $process_item, $order_items );
 
+		if ( count( $items_to_send ) > 200 ) {
+			// Stripe API max items is 200. If more than that is present, bundle everything in one "product bundle".
+			$items_to_send = [ $this->bundle_level3_data_from_items( $items_to_send ) ];
```

Comment:
I'd suggest keeping the first 199 items unchanged and grouping the rest with your approach (hiding behind 'X items more'). how does it sound?

Reply:
Same as @anu-rock 's suggestion. I think that makes sense!

I'll work on it 👍